newsrevenuehub/rev-engine
# refs #NRH-336 Adds a cleanup method for image urls.

#### What's this PR do?
Adds a cleanup method that removes google's api params for url expiration.

#### How should this be manually tested?

1. Create a DonationPage with a sidebar and images.
2. Observe that the images are served.
3. Let sit for a week or so.
4. Observe the images are still served on the DonationPage.

#### Do any changes need to be made before deployment to staging? production? (adding environment variables, for example)?
If there is a different GCloud bucket for production it will need to be configured exactly like staging especially permissions and roles.
